nion/swift/model/Symbolic.py

```
@@ -2607,6 +2616,277 @@ def recompute(self) -> typing.Optional[ComputationExecutor]:
+                self.computation_id = computation_id
+                self.computation_uuid = computation_uuid
+                self.computation_title = computation_title
+                self.output_name = output_name
+                self.computation_parameters = list(parameters)
+
+            def to_dict(self) -> dict[str, typing.Any]:
+                d = dict[str, typing.Any]()
+                if self.computation_id:
+                    d["computation_id"] = self.computation_id
+                d["computation_uuid"] = str(self.computation_uuid)
+                if self.computation_title:
+                    d["computation_title"] = self.computation_title
+                if self.output_name:
+                    d["output_name"] = self.output_name
+                d["parameters"] = [parameter.to_dict() for parameter in self.computation_parameters]
+                return d
+
+        result = self._get_output(name)
+        if result:
+            for output_item in result.output_items:
+                if isinstance(output_item, DataItem.DataItem):
+                    computation_parameters = list[ComputationRecordParameter]()
+                    for variable in self.variables:
+                        variable_specifier = variable.specifier
+                        variable_bound_item = variable.bound_item
+                        if variable_specifier and variable_bound_item:
+                            if isinstance(variable_bound_item, BoundList):
+                                list_items = list[ComputationRecordObject]()
+                                bounds_items = variable_bound_item.get_items()
+                                specifiers = variable.object_specifiers
+                                if len(bounds_items) == len(specifiers):
```

What should happen if they don't match? Presumably that is an error scenario but it gets silently ignored.

---

It gets silently ignored. Open to suggestions.